datafusion/datasource/src/file_format.rs

```diff
@@ -94,7 +93,6 @@ pub trait FileFormat: Send + Sync + fmt::Debug {
         &self,
         state: &dyn Session,
         conf: FileScanConfig,
-        filters: Option<&Arc<dyn PhysicalExpr>>,
```

Comment:
This is definitely a breaking API change, we depend on the interface intensively 😢

Reply:
Thank you @alamb, I'll add this to my  TODO list.

Reply:
Agreed. Sorry for the breakage. Do the new APIs get the job done for you, or is there something now missing?